glances/plugins/containers/engines/docker.py

```diff
@@ -257,7 +258,8 @@ def update(self, all_tag):
         try:
             # Issue #1152: Docker module doesn't export details about stopped containers
             # The Containers/all key of the configuration file should be set to True
-            containers = self.client.containers.list(all=all_tag)
+            containers = []
+            [[containers.append(container_per_client) for container_per_client in client.containers.list(all=all_tag)] for client in self.clients]
```

Comment:
Same problem as mentioned earlier, if one of the socket connections break or the server ends up serving a malformed response, then we will end up with no results.